src/Rokt-Kit.js

```diff
@@ -187,6 +198,40 @@ var constructor = function () {
     this.selectPlacements = selectPlacements;
 
     // mParticle Kit Callback Methods
+    function fetchOptimizely() {
+        var forwarders = window.mParticle
+            ._getActiveForwarders()
+            .filter(function (forwarder) {
+                return forwarder.name === 'Optimizely';
+            });
+
+        try {
+            if (forwarders.length > 0 || window.optimizely) {
+                // Get the state object
+                var optimizelyState = window.optimizely.get('state');
```

Comment:
I think you should add some handlers in case `optimizelyState` returns `undefined`.